components/ArticlesList.jsx

```diff
@@ -0,0 +1,44 @@
+import ArticleCard from "./ArticleCard";
+import { useState, useEffect } from "react";
+
+function ArticlesList() {
+
+    const [articlesArray, setArticlesArray] = useState([]);
+
+    useEffect(() => {
+        async function fetchArticles() {
```

Comment:
I would recommend taking this fetchArticles function out into an api.js file so it doesnt get too bloated here